README.md

```diff
@@ -294,8 +296,10 @@ override fun onNewIntent(intent: Intent?) {
 }
 ```
 
-**Note:** Intent handling may differ depending on your app's architecture. Adjust this example to your use-case, 
-ensuring that `Klaviyo.handlePush(intent)` is called when your app is opened from a notification.
+**Note:** Intent handling may differ depending on your app's architecture. Typically, the intent will be received in 
+your main `Activity`. If you have multiple activities, you need to call `handlePush` from any activity that could
+receive this intent. Adjust this example to your use-case, ensuring that `Klaviyo.handlePush(intent)` is called 
```

Comment:
Our default push handler uses the default launcher intent, so that should always be the main activity. But I thought we should call this out in case of someone constructing their own notification intents.